packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -58,31 +58,75 @@ type ConfigProps = {
   textColor: BaseTextProps['color']; // prop
   buttonSpacing: string; // raw value
   text?: string;
+  buttonColor: string; // raw value
+  buttonBorderColor: string; // raw value
+  hoverColor: string; // raw value
+  hoverBorderColor: string; // raw value
+  activeColor: string; // raw value
+  activeBorderColor: string; // raw value
+  focusColor: string; // raw value
+  focusBorderColor: string; // raw value
+  focusRingColor: string; // raw value
 };
 
 const getProps = ({
-  theme,
-  size,
-  children,
   buttonTypographyTokens,
+  children,
+  isDisabled,
+  size,
+  theme,
 }: {
+  buttonTypographyTokens: typeof typography.onDesktop | typeof typography.onMobile;
+  children?: string;
+  isDisabled: boolean;
   theme: Theme;
   size: BaseButtonCommonProps['size'];
-  children?: string;
-  buttonTypographyTokens: typeof typography.onDesktop | typeof typography.onMobile;
 }): ConfigProps => {
   const props: ConfigProps = {
```

Comment:
nit: `ConfigProps` sounds very generic. Let's rename this to something more specific? Maybe `BaseButtonStyleProps`?